components/FormInput/FormInput.js

```diff
@@ -28,11 +29,19 @@ export default class FormInput extends Component {
     });
   }
 
+  handleChange(e) {
```

Comment:
@iest These changes need to be made to `MaskedFormInput`, otherwise https://github.com/PactCoffee/topnotes/pull/148 breaks anything that uses that rather than `FormInput`.